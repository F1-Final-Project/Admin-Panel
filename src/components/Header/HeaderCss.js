import { makeStyles } from '@material-ui/core/styles'
import withStyles from '@material-ui/core/styles/withStyles'
import AppBar from '@material-ui/core/AppBar'
import Fab from '@material-ui/core/Fab'
import IconButton from '@material-ui/core/IconButton'
import Menu from '@material-ui/core/Menu';


export const useStyles = makeStyles(theme => ({
	root: {
		position: 'fixed',
		bottom: theme.spacing(2),
		left: '84px',
	},
	sectionDesktop: {
		display: 'none',
		[theme.breakpoints.up('md')]: {
			display: 'flex',
		},
	},
	headerToolBar: {
		display: 'flex',
		flexDirection: 'row',
		alignItems: 'center',
		justifyContent: 'flex-end',

	},
	headerLink: {
		textDecoration: 'none',
		color: '#E9C294'
	},
}))

export const CssAppBar = withStyles(theme => ({
	root: {
		backgroundColor: '#212121',
		boxShadow: '-5px -5px 10px 10px rgba(0, 0, 0, .2)',
		margin: '0 auto',
	},

}))(AppBar)
export const CssFab = withStyles(theme => ({
	secondary: {
		backgroundColor: '#d0cdc7',
		color: '#212121',

	},
	root: {
		'&:hover': {
			backgroundColor: '#E9C294',
		},
	},


}))(Fab)

export const TableIconButton = withStyles(theme => ({
	root: {
		color: '#E9C294',
		marginRight: '5px',
		'&:hover': {
			color: '#d0cdc7',
			backgroundColor: 'rgba(138, 134, 135, .4)',
			boxShadow: '7px 7px 5px 0px rgba(0,0,0,0.3)',
		},
	},
}))(IconButton)

export const CssMenuAdmin = withStyles(theme => ({
	paper: {
		backgroundColor: '#212121',
	},

}))(Menu)