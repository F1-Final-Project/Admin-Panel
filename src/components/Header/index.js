import React from 'react'

export default () => {
	return (
		<React.Fragment>
			<div>Header Test Travis CI </div>
		</React.Fragment>
	)
}
