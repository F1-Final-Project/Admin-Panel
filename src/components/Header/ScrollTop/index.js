import React from 'react'
import useScrollTrigger from '@material-ui/core/useScrollTrigger'
import { useStyles } from '../HeaderCss'
import Zoom from '@material-ui/core/Zoom'



export default function ScrollTop(props) {
	const { children, window } = props
	const classes = useStyles()

	const trigger = useScrollTrigger({ target: window ? window() : undefined })

	const handleClick = event => {

		const anchor = (event.target.ownerDocument || document).querySelector('#back-to-top-anchor')
		if (anchor) {
			anchor.scrollIntoView({ behavior: 'smooth', block: 'center' })
		}
	}

	return (
		<Zoom in={trigger}>
			<div onClick={handleClick} role="presentation" className={classes.root}>
				{children}
			</div>
		</Zoom>
	)
}