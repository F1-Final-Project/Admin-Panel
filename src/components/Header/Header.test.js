import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Header from './index'
import '../../setupTest'

describe('Header', () => {
	it('should render correctly Travis CI', () => {
		const output = shallow(<Header />);
		expect(shallowToJson(output)).toMatchSnapshot()
	})
});
