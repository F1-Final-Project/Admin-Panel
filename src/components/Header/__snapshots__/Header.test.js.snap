// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly Travis CI 1`] = `
<Fragment>
  <div>
    Header Test Travis CI 
  </div>
</Fragment>
`;
